package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ceciil-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.jsx",
+  "scripts": {
+    "build": "webpack --watch",
+    "server": "nodemon server/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/L-AirDnD/Ceciil-service.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/L-AirDnD/Ceciil-service/issues"
+  },
+  "homepage": "https://github.com/L-AirDnD/Ceciil-service#readme",
+  "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.5",
+    "eslint": "^5.13.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.12.4",
+    "webpack": "^4.29.3",
```

Comment:
Probably want to also install webpack and webpack-cli into prod dependencies as well. You did the right thing by excluding your bundle.js from your .gitignore, but that means when we push our services into our hosting service in production, there will be no bundle.js file for your app to serve. Thus, we will want our prod system to use webpack to build a final bundle.js upon postinstall.

Reply:
Great! Thanks for the feedback. I'll fix that.